translations/messages-en.properties

```diff
@@ -891,7 +891,7 @@ help.search.description = <h4>Terms</h4> <p> A query is broken up into terms and
 help.messages.title = Configuration of outgoing messages
-help.messages.description = <h4>Configuration</h4> <p>Outgoing messages can be configured to include data from the document by wrapping the field name in braces, eg\:</p> <pre>Thank you \\{\\{contact.name\\}\\}, \\{\\{patient_name\\}\\} has been registered.</pre> <p><a href\="http\://mustache.github.io/mustache.5.html" target\="_blank">More information</a>.</p> <h4>Available Fields</h4> <dl class\="horizontal"> <dt><span class\="pre">contact</span></dt> <dd>The contact at the clinic. Use contact.name or contact.phone.</dd> <dt><span class\="pre">clinic</span></dt> <dd>The clinic</dd> <dt><span class\="pre">parent</span></dt> <dd>The parent facility of the clinic</dd> <dt><span class\="pre">grandparent</span></dt> <dd>The grandparent facility of the clinic</dd> <dt><span class\="pre">health_center</span></dt> <dd>The health center</dd> <dt><span class\="pre">district</span></dt> <dd>The district</dd> <dt></dt> <dd>All other fields on the document are included</dd> </dl> <h4>Formatters</h4> <p>To format dates using the configured date format\:</p> <pre>Expected due date is \\{\\{\#date\\}\\}\\{\\{expected_date\\}\\}\\{\\{/date\\}\\}</pre> <p>Or the configured datetime format\:</p> <pre>Message received at \\{\\{\#datetime\\}\\}\\{\\{reported_date\\}\\}\\{\\{/datetime\\}\\}</pre>
+help.messages.description = <h4>Configuration</h4> <p>Outgoing messages can be configured to include data from the document by wrapping the field name in braces, eg\:</p> <pre>Thank you \\{\\{contact.name\\}\\}, \\{\\{patient_name\\}\\} has been registered.</pre> <p><a href\="http\://mustache.github.io/mustache.5.html" target\="_blank" rel\="noopener noreferrer">More information</a>.</p> <h4>Available Fields</h4> <dl class\="horizontal"> <dt><span class\="pre">contact</span></dt> <dd>The contact at the clinic. Use contact.name or contact.phone.</dd> <dt><span class\="pre">clinic</span></dt> <dd>The clinic</dd> <dt><span class\="pre">parent</span></dt> <dd>The parent facility of the clinic</dd> <dt><span class\="pre">grandparent</span></dt> <dd>The grandparent facility of the clinic</dd> <dt><span class\="pre">health_center</span></dt> <dd>The health center</dd> <dt><span class\="pre">district</span></dt> <dd>The district</dd> <dt></dt> <dd>All other fields on the document are included</dd> </dl> <h4>Formatters</h4> <p>To format dates using the configured date format\:</p> <pre>Expected due date is \\{\\{\#date\\}\\}\\{\\{expected_date\\}\\}\\{\\{/date\\}\\}</pre> <p>Or the configured datetime format\:</p> <pre>Message received at \\{\\{\#datetime\\}\\}\\{\\{reported_date\\}\\}\\{\\{/datetime\\}\\}</pre>
```

Comment:
Changing translations in the file does nothing because [it'll be overwritten next time they get exported](https://github.com/medic/medic-docs/blob/master/development/translations.md#modifying-any-existing-translation-values) from poeditor. Update the translation for all languages using poeditor, then export all to update the committed .properties files before merging this, then revert this file, then rebase this branch...